src/scss/elements/_table.scss

```diff
@@ -77,3 +77,16 @@ table:not(.life-income-gifts) {
     background:  $su-color-fog-light;
   }
 }
+
+// Wrapper that adds the horizontal scroll it the table is wider than the parent
+.table-scroll {
+  overflow: auto;
+  max-width: 100%;
+  margin-bottom: 7.2rem;
```

Comment:
This looks good to me, @Lyafel ! Only questions is - any reason why adding this big margin-bottom only on mobile?

Reply:
@yvonnetangsu I'm not. The table has spacing on deesktop, I added the spacing to the wrapper and removed it from the tablet if wrapped in table-scroll.